rel/apps/couch_epi.config

```diff
@@ -0,0 +1,9 @@
+{plugins, [
+    couch_db_epi,
+    chttpd_epi,
+    couch_index_epi,
+    global_changes_epi,
+    mango_epi,
+    mem3_epi,
+    setup_epi
```

Comment:
It is a git problem I would say.
- [Drop files/sys.config and use overlay/etc/sys.config](https://github.com/cloudant/couchdb/commit/783a8e992e84fa577c52f27624fdfed4c15d917c) renames file: `files/sys.config` -> `overlay/etc/sys.config`.
- [Configure couch_epi plugins](https://github.com/cloudant/couchdb/commit/532d3cff480a19491960d3e66e8d6df9938e0631) updates `overlay/etc/sys.config`
- [Revert "Drop files/sys.config and use overlay/etc/sys.config"](https://github.com/cloudant/couchdb/commit/ff53ff84d6fce4dc285797deb4d8840e3d2f1e40) reverts renaming of the file.
But since revert change is only a rename which is metadata update. The changes made to overlay/etc/sys.config ended up in `files/sys.config`

I'll revert [Configure couch_epi plugins](https://github.com/cloudant/couchdb/commit/532d3cff480a19491960d3e66e8d6df9938e0631) as well.